bluebanquise-disklessset/diskless/livenet_module.py

```diff
@@ -736,7 +740,7 @@ def cli_menu():
     printc('\n == Livenet image module == ', Color.GREEN)
 
     ask_module('Select an action:')
-    action_list = ['Generate a new livenet image', 'Mount an existing livenet image', 'Unount an existing livenet image', 'Resize livenet image']
+    action_list = ['Generate a new livenet image', 'Mount an existing livenet image', 'Umount an existing livenet image', 'Resize livenet image']
```

Comment:
Perhaps say `'Unmount an existing livenet image'`?

Reply:
You are right. I will fix that.